app/components/c-footer.vue

```diff
@@ -20,9 +24,13 @@
 </template>
 
 <script>
+  import cSocialMedia from '@/components/c-social-media';
+
   export default {
     name: 'c-footer',
```

Comment:
I think, according to the design the spacing on the right / left should be more than 20px

Reply:
Done